server/core/src/dataObjOpr.cpp

```diff
@@ -848,6 +849,22 @@ chkOrphanFile(
             return e.code();
         }
     }
+#else
```

Comment:
I'm assuming the #if 0 block will be deleted in a future commit

Reply:
the plan was to let the team see the old code beside the new code and verify that they are equivalent.